configuration/themes.md

```diff
@@ -5,53 +5,76 @@ description: Zed UI and syntax themes explained
 # Themes
 
 {% hint style="warning" %}
-Themes in Zed are heavily a work in progress. Themes are not guaranteed to stay the same or stay in the app as we iterate on the theme system.
+Zed themes and their underlying systems are undergoing significant development. They should not be considered stable or final. Note that themes are subject to change or removal as the theme system moves towards a stable version.
 {% endhint %}
 
-The Theme currently refers to both the UI theme and the syntax theme. These are currently linked, but in the future will likely be separated or individually customizable.
+A Zed Theme refers to and effects both the UI and the syntax highlighting of Zed.
 
-### Known issues
+We will not have formal UI and syntax themes the like Atom, but will provide a way to override syntax styles (and more) from a single theme.
 
-* Theme contrast is insufficient in places and inconsistent across the app
-* Spacing between groups and elements in places is insufficient or inconsistent
-* Semantic colors are not currently separated from syntax/accent colors (leads to some themes having awkward error colors, etc)
+---
+
+## Philosophy
+
+The theme you use in your editor is deeply personal. Whether it is focused on utterly on function, or just an asthetic that you enjoy, ultimately, a theme should be a reflection of what is imporant to you.
 
-### Themes in Zed
+We want to provide the right balance of good defaults and a powerful toolkit to make your editor your own.
 
-Come discuss themes in more depth with the team in [our community forum](https://github.com/zed-industries/community/discussions).
+In practice, this means:
 
-### Experimental themes
+- **Default Themes**: We want to provide a set of themes that are beautiful, usable and accessible out of the box. Additionally, we would love to provide default versions of prominent themes that developers are already familiar with where possible.
 
-Experimental themes are a way for us to share new themes before they are fully dialed in, or keep some themes that don't work quite right in Zed yet but people are enjoying.
+- **Powerful Customization**: We want to provide a way to customize your theme to your heart's content. This could mean taking a default theme and just changing a few colors, or it could mean creating your own theme from scratch.
 
-These themes won't get support for any issues until we have had time to update them and move them out of our experiments.
+- **Community**: We want to enable the community to create and share their themes. We will provide a way to load themes in the app, and a toolkit for building themes.
```

Comment:
```suggestion
- **Community**: We want to enable the community to create and share their themes. We will provide a way to load themes into Zed, and a toolkit for building themes.
```
Just a suggestion for wording. Clearly this comes down to personal preference. To me, `the app` confused me a bit at first.